CHANGELOG.md

```diff
@@ -25,6 +25,7 @@
 * Fix `ess/rhat` plots in `plot_forest` ([1606](https://github.com/arviz-devs/arviz/pull/1606))
 * Fix `from_numpyro` crash when importing model with `thinning=x` for `x > 1` ([1619](https://github.com/arviz-devs/arviz/pull/1619))
 * Upload updated mypy.ini in ci if mypy copilot fails ([1624](https://github.com/arviz-devs/arviz/pull/1624))
+* Enforced using `pymc3.backends.base.MultiTrace` as the `trace` value for `io_pymc3`  ([1629](https://github.com/arviz-devs/arviz/pull/1629))
```

Comment:
I would say something along the lines of raising an informative error if inferencedata is passed